aiohttp/helpers.py

```diff
@@ -34,8 +34,8 @@
 
 __all__ = ('BasicAuth', 'ChainMapProxy')
 
-PY_36 = sys.version_info >= (3, 6)
-PY_37 = sys.version_info >= (3, 7)
+PY_36 = (3, 6) <= sys.version_info < (3, 7)
```

Comment:
It is not obvious but aiohttp uses the following convention: `PY_36` means that at least `Python 3.6` is present.
It may be 3.6.3 or even 3.7
The convention makes sense for a feature support check. If the feature appears in 3.6 it is supported by all future versions.